h2o-docs/src/product/getting-data-into-h2o.rst

```diff
@@ -190,52 +192,109 @@ core-site.xml must be configured for at least the following properties (class, p
+
+- For JDBC metadata access, the Hive JDBC Driver must be on H2O job classpath.
+
 Limitations
+'''''''''''
+
+- The imported table must be stored in a :ref:`format supported by H2O<supported_file_formats>`. 
+- CSV: The Hive table property ``skip.header.line.count`` is currently not supported. CSV files with header rows will be imported with the header row as data.
+- Partitioned tables with different storage formats. H2O supports importing partitioned tables that use different storage formats for different partitions; however in some cases (for example large number of small partitions), H2O may run out of memory while importing, even though the final data would easily fit into the memory allocated to the H2O cluster.
 
-- Imported table must be stored in a format supported by H2O. (See above.)
-- CSV - Hive table property ``skip.header.line.count`` is currently not supported, CSV files with header rows will be imported with header row as data
-- Partitioned tables with different storage formats. H2O supports importing partitioned tables that use different storage formats for different partitions; however in some cases (for example large number of small partitions), H2O may run out of memory while importing even though the final data would easily fit into the memory allocated to the H2O cluster.
+Importing Examples
+''''''''''''''''''
+
+Example 1: Access Metadata via Metastore (preferred)
+####################################################
 
 .. tabs::
+
    .. code-tab:: r R
 
+        # Add the Hive JDBC driver to H2O's classpath
+        java -cp hive-jdbc.jar:<path_to_h2o_jar>: water.H2OApp
+
+        # start the h2o.jar in R
+        library(h2o)
+        h2o.init(extra_classpath=["hive-jdbc-standalone.jar"])
 
+        # basic import
         basic_import <- h2o.import_hive_table("default", "table_name")
-        multi_format_enabled <- h2o.import_hive_table("default", "table_name", allow_multi_format=True)
-        with_partition_filter <- h2o.import_hive_table("default", "table_name", [["2017", "02"]])
 
-        # access metadata via JDBC
-        basic_import <- h2o.import_hive_table("jdbc:hive2://hive-server:10000/default", "table_name")
-        # access metadata via Metastore
-        multi_format_enabled <- h2o.import_hive_table("default", "table_name", allow_multi_format=True)
-        with_partition_filter <- h2o.import_hive_table("default", "table_name", [["2017", "02"]])
+        # multi-format import
+        multi_format_enabled <- h2o.import_hive_table("default", 
+                                                      "table_name", 
+                                                      allow_multi_format=True)
 
+        # import with partitioned filter
```

Comment:
this filter actual filters which partitions are imported so I would call it a "partition filter" (not partitioned)